### PR TITLE
[Feat-699]-Change-site-and-graphics-font-to-Rubik 

### LINKED ISSUE #699: [Feat] Change site and graphics font to Rubik (Google font)

### PR DESCRIPTION
**Pull request by IsaacM890**
close #699

### CODE REVIEW

File: src/style/_fonts.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export const fontFamilyString = [
+  'Rubik',
   '"Alef"',
```
Comment (daniel-ilan) [inline src/style/_fonts.ts:3]: Do we still need Alef font?
If not, remove here and the import in index.html

**Review by daniel-ilan — APPROVED** [review]

File: src/style/_fonts.ts (hunk 1 above)
Comment (IsaacM890) [reply src/style/_fonts.ts:3]: I wasn't sure about that so I left it there , I'll ask Yuval about it , thanks .

File: public/index.html [hunk 2]
```diff
@@ -21,12 +21,8 @@
...
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <!-- Google alef font. see https://fonts.google.com/specimen/Alef -->
-    <link
-      href="https://fonts.googleapis.com/css2?family=Alef:wght@400;700&display=swap"
-      rel="stylesheet"
-      crossorigin="anonymous"
-    />
+    <!-- Google Rubik font. see https://fonts.google.com/specimen/Rubik -->
+    <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet" crossorigin="anonymous" />
```
Comment (almogbhl) [inline public/index.html:25]: we probably wish to have the font also in bold-700 as I see in the old font

File: src/style/_fonts.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 export const fontFamilyString = [
-  '"Alef"',
+  'Rubik',
```
Comment (almogbhl) [inline src/style/_fonts.ts:2]: are we sure Rubik without "" is enough, does it works locally?

File: public/index.html (hunk 2 above)
Comment (IsaacM890) [reply public/index.html:25]: I asked yuval about it , he said to wait for the designer who assigned to this issue to tell me how exactly how he want it to be  . meanwhile i put the regular font .

File: src/style/_fonts.ts (hunk 3 above)
Comment (IsaacM890) [reply src/style/_fonts.ts:2]: yes , it works

File: public/index.html (hunk 2 above)
Comment (almogbhl) [reply public/index.html:25]: I offer to add also the bold one

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**